ansible/playbooks/nagios/docs/testing_nagios_playbook.md

```diff
@@ -0,0 +1,98 @@
+## Testing The Nagios Server Playbooks - A Guide
+
+This is a brief step by step guide on how I tested the above PR... this is to prove working in a clean new vagrant environment
+
+#### 1) Identify fork & branch of code..
+
+		https://github.com/lumuchris256/infrastructure/tree/2865
+
+#### 2) Create Working Area Of Code
+
+	2.1) Clone fork
+
+		git clone https://github.com/lumuchris256/infrastructure
+
+	2.2) Switch To PR Branch
+
+		git fetch
+		git switch 2865 << This Is The Branch From The Chosen Fork >>
+
+#### 3) Create Blank Vagrant Machine To Test Server
+
+	3.1) Create a copy of the nagios vagrant file, and update with a static IP ( to make testing easier! )
+
+		cd infrastructure/ansible/playbooks/nagios
+		cp VagrantFiles/Vagrantfile.Nagios.Server.Ubuntu2204 ./Vagrantfile
+		vi Vagrantfile
+
+		<< Swap This Line >>
+		nagios_server.vm.network :private_network, type: "dhcp"
+		<< For This One >>
+		nagios_server.vm.network :private_network, ip: '192.168.50.66'
+
+	3.2) Start The Vagrant VM
+
+		vagrant up << takes a few mins >>
```

Comment:
If I just follow the commands here I end up with an authentication failure - presumably there's another step that's required?
```
==> nagios_server: Waiting for machine to boot. This may take a few minutes...
    nagios_server: SSH address: 127.0.0.1:2222
    nagios_server: SSH username: vagrant
    nagios_server: SSH auth method: private key
    nagios_server: Warning: Remote connection disconnect. Retrying...
    nagios_server: Warning: Authentication failure. Retrying...
    nagios_server: Warning: Authentication failure. Retrying...
    nagios_server: Warning: Authentication failure. Retrying...
```

Reply:
Updated the instructions for using either dynamic or static IP addressing, as using a virtual/guest network with static IPs is useful for testing nagios with multiple VMs that are being configured by the other playbooks.